gradlew

```diff
@@ -1,73 +1,129 @@
-#!/usr/bin/env bash
```

Comment:
what's the source of these gradle wrapper changes?

Reply:
By executing ./gradlew wrapper --gradle-version=7.4
See https://docs.gradle.org/7.4/release-notes.html